rfc8446.xml

```diff
@@ -159,10 +159,10 @@ are considered legacy. Those that remain all use Authenticated Encryption
 with Associated Data (AEAD) algorithms. The cipher suite concept has been
 changed to separate the authentication and key exchange mechanisms from
 the record protection algorithm (including secret key length) and a hash
-to be used with the key derivation function and the Hashed Message
-Authentication Code (HMAC).</t>
+to be used with both the key derivation function and handshake
+message authentication code.</t>
```

Comment:
While this certainly flows more nicely, isn't this hash also used (via HMAC/HKDF) in the key hierarchy?

Reply:
(Also, if we take out the HMAC expansion here, don't we end up having to expand it as part of HKDF a couple paragraphs down, which would be pretty ugly?)

Reply:
I would actually prefer to expand neither of these, just as we don't expand AES.